### ISSUE TITLE
rule: `@intlify/vue-i18n/prefer-sfc-lang-attr`

### ISSUE THREAD

**Issue by kazupon** [intlify/eslint-plugin-vue-i18n#240]
## Please describe what the rule should do:

enforce `lang` attr to be specified to custom block

## What category should the rule belong to?

- [x] Enforces code style (layout)
- [ ] Warns about a potential error (problem)
- [ ] Suggests an alternate way of doing something (suggestion)
- [ ] Other (please specify:)

## Provide 2-3 code examples that this rule should warn about:

**incorrect** code:
```vue
<i18n>
{
  "en": {
    "message": "hello!"
  }
}
</i18n>
```

**correct** code:
```vue
<i18n lang="json">
{
  "en": {
    "message": "hello!"
  }
}
</i18n>
```

## Additional context

As a background, we have a use case where we are building our own localization workflow and would like to make the format of i18n resources consistent with formatter (e.g prettier).

If i18n resources aren’t formatted, there will be a lot of diffs when syncing to Github with PR, which will make code review hard.
